robotbase/src/main/java/com/team2357/frc2022/subsystems/FeederSubsystem.java

```diff
@@ -1,21 +1,21 @@
 package com.team2357.frc2022.subsystems;
```

Comment:
We should be using a functional interface, instead of a whole Arduino here. Also this change is not relevant to the current branch, the Arduino changes should have been done on their own branch.